tests/test_blackjack_score.py

```diff
@@ -82,5 +82,25 @@ def test_returns_bust_for_scores_over_21():
   # Assert
   assert score == "Bust"
 
+#*****Consider the following two edge test cases only after you have implemented code to pass all of the tests above*****
+@pytest.mark.skip(reason="logic not yet implemented")
```

Comment:
I realize now this comment doesn't really make sense in the solution branch, but if we add this test cases to the exercise, we may want to include this comment. Does that make sense? What do you think about adding these test cases to the replit and main branch with this comment?